### PR TITLE
EZ nutriments now works properly.

### PR DESCRIPTION
**Pull request by nemvar**
## About The Pull Request

EZ nutriment now has what plants crave. It got electrolytes.

## Changelog
:cl:
tweak: EZ nutriment now contains electrolytes.
/:cl:

### CODE REVIEW

**Comment by CRITAWAKETS** [comment]
Misleading PR title.


Also powercrepe, oh my lord this is way too strong seriously can we gitban this orangeman 2.0

File: code/modules/reagents/chemistry/reagents/other_reagents.dm [hunk 1]
```diff
@@ -1306,7 +1306,7 @@
 
 /datum/reagent/plantnutriment/eznutriment
 	name = "E-Z-Nutrient"
-	description = "Cheap and extremely common type of plant nutriment."
+	description = "Contains electrolytes. It's what plants crave."
```
Comment (PKPenguin321) [inline code/modules/reagents/chemistry/reagents/other_reagents.dm]: new good

File: code/modules/reagents/chemistry/reagents/other_reagents.dm [hunk 2]
```diff
@@ -1306,7 +1306,7 @@
 
 /datum/reagent/plantnutriment/eznutriment
 	name = "E-Z-Nutrient"
-	description = "Cheap and extremely common type of plant nutriment."
```
Comment (PKPenguin321) [inline code/modules/reagents/chemistry/reagents/other_reagents.dm]: old bad

**Review by PKPenguin321 — APPROVED** [review]
Here is your review as per the Needs Review tag

**Comment by cacogen** [comment]
wtf...

**Pull request merged**